### ISSUE TITLE
Advanced compilation gives a warning

### ISSUE THREAD

**Issue by brjann** [juxt/tick#182]
When I do advanced compilation with shadow-cljs, I get this warning

```
 2 |   #?(:cljs (:require ["@js-joda/locale_en-us" :as js-joda-locale])))
   3 | 
   4 | ; doing this for the one-arity tick.core/formatter. (npm users don't get js/JSJodaLocale global automatically)
   5 | #?(:cljs (set! js/JSJodaLocale js-joda-locale))
---------------------^----------------------------------------------------------
 constant JSJodaLocale assigned a value more than once.
Original definition at externs.shadow.js:4
```

According to Thomas Heller, it should be ` (js/goog.exportSymbol "JSJodaLocale" js-joda-locale)` instead (see https://clojurians.slack.com/archives/C6N245JGG/p1678273154348659)

**Comment by henryw374** [comment]
as it is currently, it works. but yes getting rid of warning would be good. 

either what you suggest would work or possibly `(goog.object/set js/global "JSJodaLocale" js-joda-locale)` - which may be equivalent.

**Comment by henryw374** [comment]
was fixed in 0.6.2
